src/main/java/com/cosmian/jna/findex/structs/NextKeyword.java

```diff
@@ -0,0 +1,25 @@
+package com.cosmian.jna.findex.structs;
+
+import java.nio.charset.Charset;
+
+import com.cosmian.jna.findex.serde.Leb128ByteArray;
+
+public class NextKeyword extends Leb128ByteArray {
+
+    public NextKeyword() {
+        super();
+    }
+
+    public NextKeyword(byte[] keyword) {
+        super(keyword);
+    }
+
+    public NextKeyword(String keyword) {
+        this(keyword.getBytes(Charset.defaultCharset()));
```

Comment:
In `Keyword` you use `StandardCharsets.UTF_8`, which method is best? Being explicit about UTF8?

Reply:
This is a bug. Fixed. TXS